pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -276,7 +276,7 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 	// If the cluster has Multus enabled we need to append the network namespace of the driver's
 	// holder DaemonSet in the csi configmap
 	if cephCluster.Spec.Network.IsMultus() {
-		netNamespaceFilePath, err := csi.GenerateNetNamespaceFilePath(r.opManagerContext, r.client, cephCluster.ClusterName, r.opConfig.OperatorNamespace, csi.CephFSDriverShortName)
+		netNamespaceFilePath, err := csi.GenerateNetNamespaceFilePath(r.opManagerContext, r.client, cephCluster.Name, r.opConfig.OperatorNamespace, csi.CephFSDriverShortName)
```

Comment:
@leseb The ClusterName field is no longer available. This net namespace file path is intended to be unique across K8s clusters using the same nodes? Or is unique across ceph clusters sufficient and we can just use the namespace? Will it affect upgrades if this value changes from ClusterName to namespace or something else?